cloudigrade/api/clouds/aws/tasks/verification.py

```diff
@@ -23,45 +21,23 @@ def verify_account_permissions(account_arn):
     Args:
         account_arn: The ARN for the account to check
 
-    Returns:
```

Comment:
Should this have stayed? Since line 40 does seem to still return a bool.

Reply:
Good catch. I've restored the "Returns" section and rewritten the whole docstring with (hopefully) better wording in general.